src/components/Elements/OpenGraphImage/index.tsx

```diff
@@ -0,0 +1,138 @@
+import { loadDefaultJapaneseParser } from "budoux"
+import fs from "fs"
```

Comment:
個人的には、`node:`プレフィックスを付けておいた方が良いと思うのですが、どうでしょうか？

理由：`node:test`などプレフィックス必須のモジュールとの相性が良く、またNode.jsのAPIであることが明示できる

> 参考文献：
> https://zenn.dev/yogarasu/articles/03e8b97519c17e
> https://qiita.com/ohakutsu/items/b70c1ccf2f9fb30cd048

```suggestion
import fs from "node:fs"
```